react-ui/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import LogIn from "./components/Layouts/LogIn"
+import Dashboard from "./components/Layouts/Dashboard"
+import './App.css'
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect
+} from "react-router-dom";
+
+
+class App extends Component {
+  state = {
+    hello: null,
+    postgres: null,
+    error: null
+  }
+
+  componentDidMount() {
+    fetch('/api/hello')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ hello: data })
+      })
+
+    fetch('/api/postgres')
```

Comment:
It will always throw an error. We currently don't have an endpoint for `/api/postgres`. Is it possible to change this to `/test`?

Reply:
I believe this was your code that I cloned this from the old repo. It is probably obsolete by now as it was in the very early stages of the repo - should I delete lines 20 to 42? @catai9

Reply:
Sure, just make sure that it doesn't generate any errors. I think it was more of a template to show how one would fetch from the backend.